### PR TITLE
SWC-5312

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep2.java [hunk 1]
```diff
@@ -101,6 +101,7 @@ public void configure(ManagedACTAccessRequirement accessRequirement) {
 		view.setIsIDUPublic(accessRequirement.getIsIDUPublic());
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep2.java:101]: ACT UI for creating a Managed AR.  Initialize to AR config, and update from view on save.

File: src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep2ViewImpl.java [hunk 2]
```diff
@@ -46,11 +48,11 @@
...
-			@Override
-			public void onClick(ClickEvent event) {
-				presenter.onEditWiki();
-			}
+		editWikiButton.addClickHandler(event -> {
+			presenter.onEditWiki();
+		});
+		iduCheckbox.addValueChangeHandler(event -> {
+			intendedDataUsePublicCheckbox.setEnabled(event.getValue());
 		});
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep2ViewImpl.java:56]: When the IDU required checkbox is clicked, update enable state of the "is IDU public" checkbox (not applicable if IDU is optional).  Is also updated below, when it's initialized from an existing AR.

File: src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/requestaccess/CreateResearchProjectStep1.java [hunk 3]
```diff
@@ -44,6 +44,7 @@ public CreateResearchProjectStep1(CreateResearchProjectWizardStep1View view, Dat
 	public void configure(ManagedACTAccessRequirement ar, RestrictableObjectDescriptor targetSubject) {
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/requestaccess/CreateResearchProjectStep1.java:44]: User-facing requestor UI.  For now, only show the IDU text box if an IDU is required.  And don't try to validate the IDU value if it's optional.

File: src/main/webapp/sass/swc.scss [hunk 4]
```diff
@@ -34,12 +34,13 @@ a {color:$synapse-blue;text-decoration:none;}
 a:visited{color:$synapse-blue; text-decoration:none;}
 a:hover{color:$synapse-blue; text-decoration:underline;}
 a:focus{color:$synapse-blue; text-decoration:underline;}
```
Comment (jay-hodgson) [inline src/main/webapp/sass/swc.scss:36]: Lightened up color value when elements were not enabled (to make it more obvious in the UI).  Also apply to checkbox and radio "disabled" state.

**Review by xschildw — APPROVED** [review]

**Pull request merged**